### PR TITLE
treewide: replace bazaar with breezy

### LINKED ISSUE #80740: Replace Bazaar with Breezy
**Issue description**
[Bazaar](http://wiki.bazaar.canonical.com/Download) is in maintenance mode and stuck at python 2.7, which is now officially unsupported.
A fork called [Breezy](https://www.breezy-vcs.org/), which is backward compatible and supports python 3, has been available for a while and should replace it.

**Possible course of action**

1. Package Breezy in nixpkgs
2. Alias `bazaar` to `breezy` and test dependent packages
3. Replace `bazaar` with `breezy` for working packages
4. Remove `bazaar` and broken packages (20.09)

**Affected packaged**
- fetchbzr ✓
- qarte ✓
- reposurgeon (broken on master)
- cvs2svn ✓
- ikiwiki ✓
- **nix-prefetch-scripts** ✓
- vcstool ✓
- flatpak-builder ✓
- **hydra** ✓
- ~PyLD~
- ~powerline~
- gsettings-qt ✓
- ~libftdi~
- ~pmount~
- ~mod_dnssd~

**Affected NixOS modules**
- redmine (broken on master #80061)

---

✓: works with breezy
✗: doesn't work
~--~: false positive

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
Fix #80740 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change (see below)
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (see below)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### nix-review report

Result of `nixpkgs-review` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 package built:</summary>

  - cabal2nix
  - dep2nix
  - flatpak-builder
  - hydra-migration
  - hydra-unstable
  - nix-prefetch-bzr
  - nix-prefetch-scripts
  - nix-update-source
  - vcstool
</details>

###### closure size increase

As @jtojnar noticed, breezy constitues a large increase in closure size due to an issue in `python3Packages.secretstorage`. The solution is trivial but I decided to deal with this in another PR not to mix up the two changes in the results of nix-review.

edit: Fixed in #85571

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -50,6 +50,7 @@ mapAliases ({
   bar-xft = lemonbar-xft;  # added 2015-01-16
   bashCompletion = bash-completion; # Added 2016-09-28
   batti = throw "batti has been removed from nixpkgs, as it was unmaintained"; # added 2019-12-10
+  bazaar = throw "bazaar has been deprecated by breezy."; # added 2020-04-19;
```
Comment (prusnak) [inline pkgs/top-level/aliases.nix:53]: a superfluous semicolon at the end of the line

Comment (rnhmjoj) [reply pkgs/top-level/aliases.nix]: Thank you, fixed.

**Review by jtojnar — APPROVED** [review]

**Comment by rnhmjoj** [comment]
I would like to get some feedback from the maintainers, if possible, before going on:

- @aanderse for redmine?
- @peti for iwikiwi
- @makefu for cvs2svn
- @dfoxfranke for reposurgeon
- @ma27 for hydra
- @sivteck for vcstool

**Review by makefu — APPROVED** [review]
I approve of this PR, less python2-only packages is a plus.
csv2svn only uses bazaar as `checkInput` not much to stop working here as long as the tests succeed.
It would probably a good idea to add the deprecation notice into the release-notes and maybe even add some documentation to the manual, however i would not even know where to put it

**Comment by rnhmjoj** [comment]
> It would probably a good idea to add the deprecation notice into the release-notes and maybe even add some documentation to the manual, however i would not even know where to put it

Right, I now mentioned the change in the release notes. Thank you.

**Review by Ma27 — APPROVED** [review]

**Comment by rnhmjoj** [comment]
Rebased. I think it has received enough feedback, now: I'm going to merge.

**Pull request merged**